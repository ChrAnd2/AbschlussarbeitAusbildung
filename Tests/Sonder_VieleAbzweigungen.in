//******************************************
//+ Sonder_VieleAbzweigungen- Jeder zu Jedem ohne Zyklus
//******************************************
//Vorgangsnummer; Vorgangsbezeichnung; Dauer; Vorgänger; Nachfolger
1; A; 1; -; 2,3,4,5,6,7,8,9,10
2; B; 1; 1,11,12,13,14,15,16,17,18; -
3; C; 1; 1,11,12,13,14,15,16,17,18; -
4; D; 1; 1,11,12,13,14,15,16,17,18; -
5; E; 1; 1,11,12,13,14,15,16,17,18; -
6; F; 1; 1,11,12,13,14,15,16,17,18; -
7; G; 1; 1,11,12,13,14,15,16,17,18; -
8; H; 1; 1,11,12,13,14,15,16,17,18; -
9; I; 1; 1,11,12,13,14,15,16,17,18; -
10; J; 1; 1, 11,12,13,14,15,16,17,18; - 
11; K; 1; -; 2,3,4,5,6,7,8,9,10
12; L; 1; -; 2,3,4,5,6,7,8,9,10
13; M; 1; -; 2,3,4,5,6,7,8,9,10
14; O; 1; -; 2,3,4,5,6,7,8,9,10
15; P; 1; -; 2,3,4,5,6,7,8,9,10
16; Q; 1; -; 2,3,4,5,6,7,8,9,10
17; R; 1; -; 2,3,4,5,6,7,8,9,10
18; S; 1; -; 2,3,4,5,6,7,8,9,10