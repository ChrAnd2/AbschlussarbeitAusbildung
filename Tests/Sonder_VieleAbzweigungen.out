Sonder_VieleAbzweigungen- Jeder zu Jedem ohne Zyklus

Vorgangsnummer; Vorgangsbezeichnung; D; FAZ; FEZ; SAZ; SEZ; GP; FP
1; A; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
2; B; 1.0; 1.0; 2.0; 1.0; 2.0; 0.0; 0.0
3; C; 1.0; 1.0; 2.0; 1.0; 2.0; 0.0; 0.0
4; D; 1.0; 1.0; 2.0; 1.0; 2.0; 0.0; 0.0
5; E; 1.0; 1.0; 2.0; 1.0; 2.0; 0.0; 0.0
6; F; 1.0; 1.0; 2.0; 1.0; 2.0; 0.0; 0.0
7; G; 1.0; 1.0; 2.0; 1.0; 2.0; 0.0; 0.0
8; H; 1.0; 1.0; 2.0; 1.0; 2.0; 0.0; 0.0
9; I; 1.0; 1.0; 2.0; 1.0; 2.0; 0.0; 0.0
10; J; 1.0; 1.0; 2.0; 1.0; 2.0; 0.0; 0.0
11; K; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
12; L; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
13; M; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
14; O; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
15; P; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
16; Q; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
17; R; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
18; S; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0

Anfangsvorgang: 1,11,12,13,14,15,16,17,18
Endvorgang: 2,3,4,5,6,7,8,9,10
Maximale Gesamtdauer (da mehrere Endzustaende): 2.0

Kritische Pfade:
1. 1->2
2. 1->3
3. 1->4
4. 1->5
5. 1->6
6. 1->7
7. 1->8
8. 1->9
9. 1->10
10. 11->2
11. 11->3
12. 11->4
13. 11->5
14. 11->6
15. 11->7
16. 11->8
17. 11->9
18. 11->10
19. 12->2
20. 12->3
21. 12->4
22. 12->5
23. 12->6
24. 12->7
25. 12->8
26. 12->9
27. 12->10
28. 13->2
29. 13->3
30. 13->4
31. 13->5
32. 13->6
33. 13->7
34. 13->8
35. 13->9
36. 13->10
37. 14->2
38. 14->3
39. 14->4
40. 14->5
41. 14->6
42. 14->7
43. 14->8
44. 14->9
45. 14->10
46. 15->2
47. 15->3
48. 15->4
49. 15->5
50. 15->6
51. 15->7
52. 15->8
53. 15->9
54. 15->10
55. 16->2
56. 16->3
57. 16->4
58. 16->5
59. 16->6
60. 16->7
61. 16->8
62. 16->9
63. 16->10
64. 17->2
65. 17->3
66. 17->4
67. 17->5
68. 17->6
69. 17->7
70. 17->8
71. 17->9
72. 17->10
73. 18->2
74. 18->3
75. 18->4
76. 18->5
77. 18->6
78. 18->7
79. 18->8
80. 18->9
81. 18->10
